src/main/java/com/df/Onboarding/phone/controller/PhoneController.java

```
@@ -20,12 +20,14 @@ public void addNumber(@RequestBody PhoneNumber phoneNumber){
 
 
     @GetMapping("/phone/")
-    public String requestNumbers(@RequestParam int id) {
+    public PhoneNumber requestNumbers(@RequestParam int id) {
         Optional<PhoneNumber> phoneNumber = phoneNumberRepo.findById(id);
         if (phoneNumber.isPresent()) {
-            return phoneNumber.get().getPhoneNumber();
+            return phoneNumber.get();
         } else {
-            return "";
```

You can throw an exception here, and handle it globally